tests/base/fn2/requirements.txt

```diff
@@ -1 +1 @@
-dataclasses
\ No newline at end of file
+dataclasses
```

Comment:
hmm.. what actually was changed here? 

Is it a GitHub quirk?

Reply:
I didn't notice, it's a change about missing empty line at the end of file, I've reverted it and I'll merge after tests pass

Reply:
Ah.. it was fine by all means. I was just puzzled